scanpipe/models.py

```diff
@@ -1674,6 +1674,11 @@ def status(self, status=None):
             return self.filter(status=status)
         return self.filter(~Q(status=""))
 
+    def not_status(self, status=None):
+        if status:
+            return self.filter(~Q(status=status))
+        return self.filter(status="")
+
     def no_status(self):
         return self.filter(status="")
 
```

Comment:
Could we merge both functions to avoid redundancy? Maybe add the status option to the existing `no_status`.